quodlibet/tests/test_formats__audio.py

```diff
@@ -408,6 +408,117 @@ def test_lyric_filename(self):
         song["lyricist"] = u"Lyricist"
         self.assertTrue(isinstance(song.lyric_filename, fsnative))
 
+    def test_lyric_filename_search(self):
+
+        def assertEqual(a, b):
+            # don't truncate the bloody error message..
+            msg_fail = str("item1: %s !=\nitem2: %s" % (a, b))
+            self.assertEqual(a, b, msg_fail)
+
+        with temp_filename() as filename:
```

Comment:
These all seem like great _independent_ tests. Some people use the rule of "one assertion per test" - bit strict IMO but the comments here are good indicators that there are separate conditions being tests => separate tests. If there's code common to each of them you can use methods, `setUp`, decorators or helpers like some of the other tests...